### PR TITLE
API doc improvements

### PR DESCRIPTION
**Pull request by sclu1034**
Various accumulated bits.

### CODE REVIEW

**Review by actionless — APPROVED** [review]

**Review by Aire-One — APPROVED** [review]
👍

**Comment by Aire-One** [comment]
Oops... I'm not sure why Travis failed with:

```
Failure -> spec/beautiful/init_spec.lua @ 14

beautiful init Check beautiful.get_font

spec/beautiful/init_spec.lua:15: Expected objects to be the same.

Passed in:

(userdata) 'lgi.rec 0x1eea500:Pango.FontDescription'

Expected:

(userdata) 'lgi.rec 0x155c580:Pango.FontDescription'

stack traceback:

	spec/beautiful/init_spec.lua:15: in function <spec/beautiful/init_spec.lua:14>
```

**Comment by actionless** [comment]
the test sees to be always broken, idk why it even was working before:

it should be pango.FontDescription.to_string (https://developer.gnome.org/pygtk/stable/class-pangofontdescription.html)

File: docs/90-FAQ.md
Comment (actionless) [inline docs/90-FAQ.md:64]: why not just `sudo curl ... -o ...`?

Comment (sclu1034) [reply docs/90-FAQ.md:64]: Since the display manager will execute whatever the `Exec` line will specify, this is awfully close to `curl ... | sudo bash`.
So I chose a command that, at the very least, prints the file's content for verification.

**Comment by actionless** [comment]
i've submitted a fix for that test: https://github.com/awesomeWM/awesome/pull/3348

File: docs/90-FAQ.md
Comment (actionless) [reply docs/90-FAQ.md:64]: ok

**Review by Aire-One — APPROVED** [review]
I guess we'll have to wait for #3348 to be merged before, tho...

**Comment by sclu1034** [comment]
@Aire-One I wouldn't mind waiting, but I can't follow why that would be necessary.
This PR doesn't change functionality, so whether that one test passes isn't affected by this change.

And Travis did pass after the latest push, now it's just code coverage being weird about a line that moved but didn't change.

**Comment by Aire-One** [comment]
I suspected Travis wouldn't pass without the fix, and we would have been stuck :shrug: This is very unclear to me the test failed the run before and now work...

I can see that I have a "re-run" button next to the codecov result in the check tab. Let's see what happen if I click it...

We'll need a second approver to merge this PR anyway.

**Review by Elv13 — APPROVED** [review]

**Pull request merged**
